projects/components/src/dropdown/dropdown-focus-handler.service.ts

```diff
@@ -23,41 +23,47 @@ export class DropdownFocusHandlerService implements OnDestroy {
     }
 
     /**
-     * Moves the focus to HTML element in the given direction
+     * Moves the focus to HTML element in the given direction and returns true if the focus is moved and false otherwise
      */
-    moveFocus(direction: Direction): void {
+    moveFocus(direction: Direction): boolean {
+        let moved;
         switch (direction) {
             case Direction.DOWN:
-                this.moveFocusTo(this.currentFocusedItem.down);
+                moved = this.moveFocusTo(this.currentFocusedItem.down);
                 break;
             case Direction.LEFT:
-                this.moveFocusTo(this.currentFocusedItem.left);
+                moved = this.moveFocusTo(this.currentFocusedItem.left);
                 break;
             case Direction.UP:
-                this.moveFocusTo(this.currentFocusedItem.up);
+                moved = this.moveFocusTo(this.currentFocusedItem.up);
                 break;
             case Direction.RIGHT:
-                this.moveFocusTo(this.currentFocusedItem.right);
+                moved = this.moveFocusTo(this.currentFocusedItem.right);
                 break;
         }
+        return moved;
     }
 
     /**
-     * Calls the HTML focus method on the HTML element passed
+     * Calls the HTML focus method on the HTML element passed and removed focus on currently focused element.
+     * Returns true if the focus is moved and false otherwise.
      */
-    moveFocusTo(item: MenuItem): void {
+    moveFocusTo(item: MenuItem): boolean {
+        let moved = false;
```

Comment:
Instead of having the moved variable, cant we just return a specific boolean from the places where we `return moved`. So the first two would be `return false` and the last one would be `return true`?